### DISCUSSION TITLE
CBOR for IoT Identity Devices

### DISCUSSION THREAD

**Discussion by paulgrehan** [decentralized-identity/SIG-IoT#8] [Ideas]
Keen to discuss leveraging CBOR for use in constrained environments, Gordian Envelope work looks interesting but CBOR in general seems to be the right fit for most smaller footprint devices that are purpose built for other primary functions (ie. not much CPU/memory or key space for DIDs/VCs)

**Comment by InternazionaleMilano1908** [comment]
By using CBOR, IoT devices can transmit and receive data more efficiently, which can help reduce data usage, improve performance, and reduce costs.
